### PR TITLE
feat(grafana): RHICOMPL-3350 opt-exclude IQE stats from counters

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: dashboards/grafana-dashboard-insights-compliance-general.template.yaml [hunk 1]
```diff
@@ -133,7 +133,7 @@ objects:
                 "datasource": {
                   "uid": "$datasource"
                 },
-                "expr": "sum(increase(rails_requests_total{application=\"compliance\"}[$__range]))",
+                "expr": "sum(increase(rails_requests_total{application=\"compliance\"}[$__range])) * $qe + sum(increase(rails_requests_total{application=\"compliance\",org_id!~\"${QE_ACCOUNTS}\"}[$__range])) * (-($qe - 1))",
```
Comment (vkrizan) [inline dashboards/grafana-dashboard-insights-compliance-general.template.yaml:136]: The `org_id` matches are lacking exact match.

Comment (skateman) [reply dashboards/grafana-dashboard-insights-compliance-general.template.yaml:136]: It is intentional, we need the regex as we have to match against multiple org_ids, like `123|456|789` but I could adjust it on the other side as `^123|456|789$` if that is enough...

Comment (vkrizan) [reply dashboards/grafana-dashboard-insights-compliance-general.template.yaml:136]: As per Prometheus docs:
> Regex matches are fully anchored. A match of env=~"foo" is treated as env=~"^foo$".

Disregard.

**Review by vkrizan — APPROVED** [review]

**Pull request merged**